From: SeongJae Park <sj@kernel.org>
Date: Mon, 8 Sep 2025 18:32:31 -0700
Subject: [PATCH] ==== kunut improvements ====

mm/damon/tests: add more tests for online parameters commit

A DAMON feature called parameters "commit" allows DAMON API callers and
ABI users update nearly every DAMON parameter while DAMON is running.
This is being used for flexible DAMON use cases such as taking snapshot
of the monitoring results with minimum overhead, or adjusting
access-aware system operations (DAMOS) for user-space driven auto-tuning
or investigations.

Compared to the usefulness of the feature and size of the
implementation, the test coverage is pretty small.  Only filter commit
part has a single test case, namely damos_test_commit_filter().
Actually, we found and fixed a few bugs of the feature in the past.  The
single existing test was also added to avoid reintroduce of a found bug.
Add more unit tests for the feature.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 hkml_cv_bogus/hkml_cv_bogus_jol40ka9 | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 hkml_cv_bogus/hkml_cv_bogus_jol40ka9

diff --git a/hkml_cv_bogus/hkml_cv_bogus_jol40ka9 b/hkml_cv_bogus/hkml_cv_bogus_jol40ka9
new file mode 100644
index 000000000000..e69de29bb2d1
-- 
2.47.3

